### ISSUE TITLE
Crash with square brackets in security_group_id in aws_db_security_group

### ISSUE THREAD

**Issue by buddhabuddy** [hashicorp/terraform#956]
I had the following which caused a crash:

    resource "aws_db_security_group" "rds" {
        name = "rds-sg"
        description = "rds"

        ingress {
          from_port = 5432
          to_port = 5432
          protocol = "tcp"
          security_group_id = ["${aws_security_group.app.id}"]
        }
    }

removing the square brackets sorted it.

**Comment by ceh** [comment]
Doesn't crash on `Terraform v0.3.7-dev (3339593292d2d16922cbf6270fbbb2bfea7d6440)`:

```
There are warnings and/or errors related to your configuration. Please
fix these before continuing.

Errors:

  * 'aws_db_security_group.rds' error: '' expected type 'string', got unconvertible type '[]interface {}'
```

**Comment by buddhabuddy** [comment]
Sweet, that's it fixed in v0.3.7 then.
